microservices/optimizer/app.py

```diff
@@ -1,12 +1,20 @@
 """Optimizer: the optimization microservice. More information can be found
 in the documentation at `optimizer_api_doc.md`"""
 
-from flask import Flask, jsonify, Response, request
+from flask import Flask, jsonify, Response, request, g
 
-from utils import optimize
+from utils import DBManager, optimize
 
 app = Flask(__name__)
 
+DATABASE_FILE: str = "opt.db"
```

Comment:
The database is off service so there is no file to use (I've updated first to fix this in my Kubernetes PR)
```suggestion

```